### ISSUE TITLE
Change username length requirment

### ISSUE THREAD

**Issue by dcnadler** [jupyterhub/jupyterhub#4015]
### Proposed change
Change the min length of username from 3 to 2: https://github.com/jupyterhub/jupyterhub/blob/main/jsx/src/components/EditUser/EditUser.jsx#L130

Our organization uses last name as usernames and people with two-letter last names cannot be added through the front-end.

### Alternative options
None

### Who would use this feature?
Those who want 2-letter usernames

**Comment by minrk** [comment]
Just to be clear, did you only encounter this issue editing users via the admin UI? JupyterHub doesn't prohibit 2-character usernames in general.

**Comment by minrk** [comment]
The client-side validation should all be removed, I think #4016 finishes that.

**Comment by dcnadler** [comment]
> Just to be clear, did you only encounter this issue editing users via the admin UI? JupyterHub doesn't prohibit 2-character usernames in general.

Correct. We can add users with 2-letter usernames fine with `jupyterhub_config.py`, it was just the front-end that prevented it.

Thanks for the fix!
